extras/kinesis-asl/src/test/java/org/apache/spark/streaming/kinesis/JavaKinesisStreamSuite.java

```diff
@@ -34,7 +34,7 @@ public void testKinesisStream() {
     // Tests the API, does not actually test data receiving
     JavaDStream<byte[]> kinesisStream = KinesisUtils.createStream(ssc, "mySparkStream",
         "https://kinesis.us-west-2.amazonaws.com", new Duration(2000), 
-        InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2());
+        InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2(), null);
```

Comment:
would be good to have tests around setting the credentials in the following 2 cases:
1) None (null)
2) Some[AWSCredentials]